ui/component/claimList/view.jsx

```diff
@@ -33,7 +33,7 @@ type Props = {
   defaultSort?: boolean,
   onScrollBottom?: (any) => void,
   page?: number,
-  pageSize?: number,
+  pageSize: number,
```

Comment:
`pageSize` should remain optional, I believe. There are lots of `<ClaimList>` instances without `pageSize` and it works.

Reply:
The usual fix is to make the code handle the undefined case (skip the loop or something), instead of the other way around of changing the definition to make it pass.  The warning is to remind us of all possible code paths.

Example existing code does things only when page and pageSize is defined `if (page && pageSize && onScrollBottom) {`